core/src/test/java/org/opensearch/sql/analysis/ExpressionAnalyzerTest.java

```diff
@@ -342,6 +343,11 @@ public void named_non_parse_expression() {
     assertAnalyzeEqual(DSL.ref("string_field", STRING), qualifiedName("string_field"));
   }
 
+  @Test
+  public void match_phrase_prefix_all_parameters() {
+
+  }
+
```

Comment:
The test is implemented in `match_phrase_prefix_all_params` below in this file, isn't it?